man/split.Rd

```diff
@@ -47,6 +47,9 @@ all.equal(
 split(DT, by=c("x1", "x2"))
 split(DT, by=c("x1", "x2"), flatten=FALSE)
 
+# use formula instead of by
```

Comment:
Since we discourage `f=`, I will remove this example. We can restore in follow-up if you feel strongly.